package.json

```diff
@@ -14,30 +20,33 @@
     "@angular/platform-browser-dynamic": "^8.0.0",
     "@types/jasmine": "^3.3.13",
     "awesome-typescript-loader": "^5.2.1",
-    "copyfiles": "^2.1.0",
+    "codelyzer": "^5.0.1",
     "core-js": "^2.6.9",
-    "del-cli": "^2.0.0",
     "jasmine-core": "^3.4.0",
+    "jasmine-spec-reporter": "^4.2.1",
     "karma": "^4.1.0",
+    "karma-chrome-launcher": "^2.2.0",
+    "karma-coverage-istanbul-reporter": "^2.0.5",
     "karma-jasmine": "^2.0.1",
-    "karma-phantomjs-launcher": "^1.0.4",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^3.0.5",
-    "phantomjs-prebuilt": "^2.1.16",
+    "karma-jasmine-html-reporter": "^1.4.2",
+    "ng-packagr": "^5.1.0",
     "rxjs": "^6.5.2",
+    "serve": "^11.0.1",
+    "svg-sprite": "^1.5.0",
+    "tsickle": "^0.35.0",
+    "tslib": "^1.9.0",
     "tslint": "^5.17.0",
     "typescript": "3.4.5",
     "webpack": "^4.33.0",
-    "zone.js": "^0.9.1",
-    "@angular-devkit/build-angular": "~0.800.1"
+    "zone.js": "^0.9.1"
   },
   "scripts": {
-    "start": "./node_modules/.bin/ng serve",
```

Comment:
this should not be removed

Reply:
I don't think the `./node_modules/.bin` prefix is needed.. npm resolves all binaries inside of the `.bin` automatically.. at least if you call `npm start` or `npm run start` it works.. do you still want me to change this?